README.md

```diff
@@ -69,12 +69,29 @@ Language: You must have GoLang installed to run Scorecards (https://golang.org/d
 
 ### Installation
 
-To install Scorecards:
+#### Standalone
+
+To install Scorecards as a standalone:
 
 1. Visit our latest [release page](https://github.com/ossf/scorecard/releases/latest) and download the correct binary for your operating system
 2. Extract the binary file 
 3. Add the binary to your `GOPATH/bin` directory (use `go env GOPATH` to identify your directory if necessary)
 
+#### Using Homebrew
+
+You can use [Homebrew](https://brew.sh/) (on macOS or Linux) to install Scorecards.
+
+```sh
+brew install scorecard
+```
+
+### Using Linux package managers
+
+| Package Manager                                            | Linux Distribution | Command                                    |
+|------------------------------------------------------------|--------------------|--------------------------------------------|
+| Nix                                                        | NixOS              | `nix-env -iA nixpkgs.scorecard`            |
```

Comment:
Same question here. We don't have this setup AFAIK.

Reply:
It was submitted to nixpkgs[^1] by @06kellyjac

[^1]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/tools/security/scorecard/default.nix

Reply:
Can we add this is maintained by the community https://github.com/NixOS/nixpkgs/blob/master/pkgs/tools/security/scorecard/default.nix? This will help with the expectations.

Reply:
pretty much any package via a package manager is maintained by the "community", excluding the big businesses such as RedHat, OpenSUSE, Canonical

unless if you mean the distinction between the official arch repositories and the community AUR or the official brew repositories and separate casks. If that's the distinction we're using then this is an "official" package in the official nixpkgs, the AUR equivalent in nix/nixpkgs is the NUR.

Reply:
Thanks @06kellyjac!